### DISCUSSION TITLE
v45.0-beta1

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#4157] [Release Discussions]
<h3>🍰 Overlays!</h3>
  <p>Make every survey unique! How? Dedicate each to updating a different aspect of the map by activating a different overlay, respectively!</p>
  <p>This is <b>for advanced users</b> who want to be more efficient in adding and maintaining certain data area-wide. Read more in <a href="https://www.openstreetmap.org/user/westnordost/diary/399378">this blog post</a>!</p>
  <p>For now, there are only overlays for <i>street lighting</i> and for <i>sidewalks</i>, more will be added over time. 🎉 (#2461, #2506, #3920, #3644)</p>
  <h3>New Quests</h3>
  <ul>
  <li><i>What kind of memorial is this?</i> (#4102, #1572), by @matkoniecz, thanks @kmpoppe, @naposm, @FloEdelmann</li>
  <li><i>Is there a summit cross here?</i> (#4111, #4095), by @FloEdelmann</li>
  <li><i>What is signed at this entrance?</i> (#4066, #3064), by @matkoniecz</li>
  </ul>
  <h3>Enhancements and Fixes</h3>
  <ul>
  <li>Do not assume that any cycle lane or track on the right side of the road will always be in forward direction (in countries with right-hand traffic, for left-hand traffic respectively)</li>
  <li>Do not require OpenStreetMap read/write GPS track permission for login (the record GPS track feature can then simply not be used) (#4122)</li>
  <li>Improve GPS accuracy for some devices (#34139...), by @Helium314</li>
  <li>In dark mode, remove obstrusive glow from railways and motorways, by @Helium314</li>
  <li>Other small enhancements and performance improvements (#3919, #4120...)</li>
  </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/streetcomplete/StreetComplete/releases/tag/v45.0-beta1'>v45.0-beta1</a>.</em>

**Comment by westnordost** [comment]
I didn't mention all the bugfixes from v45.0-alpha1 because this changelog is for users of other public beta versions but safe to say there were some, including a critical one for the shoulder quest (what was displayed was not what was tagged, or at least it was highly confusing).

**Comment by deevroman** [comment]
Notes for the quest "What is signed at the entrance?"

1. Maybe show open the keyboard right away?

https://user-images.githubusercontent.com/25708359/175819951-501106d0-49c3-4451-8f57-a520826fea18.mp4


3. How about the Ok button goes to the next field?

https://user-images.githubusercontent.com/25708359/175819955-c61bca56-80ee-4b0b-a0e2-c8711a990dac.mp4

**Reply by matkoniecz** [reply]
ad 1) https://github.com/streetcomplete/StreetComplete/issues/2131#issuecomment-703248678

ad 3) ops, this was mentioned I think at some point and lost - can you open an issue for that? (also "to" is getting malformed here - what is resolution or model of your phone? This way I should be able to reproduce it in an emulator).

**Reply by deevroman** [reply]
#4160

**Comment by Jean-BaptisteC** [comment]
With v45.0-beta1, when you actualize quests, all quests (enabled and disabled quests) are show on map even quest selection is by default.
If you open app settings, others fragments or leave app, quest disabled are disappear.

**Reply by deevroman** [reply]
#4159

**Comment by thibaultmol** [comment]
1) is it intentional that the sidewalks overlay gets obscured by buildings? It makes it kinda hard to use though.
2) in the lit overlay: have the last selected option be selected by default (similar to other quests)

**Reply by westnordost** [reply]
> in the lit overlay: have the last selected option be selected by default (similar to other quests)

In general, overlays **always** show how it is currently mapped. If you tap on a road that has been tagged as lit, than it is preselected as "lit". (Also, no quests select the previous option by default)

**Reply by mnalis** [reply]
>> 1. is it intentional that the sidewalks overlay gets obscured by buildings? It makes it kinda hard to use though.

Are you talking about 3D display of the building being problematic (e.g. like in https://github.com/streetcomplete/StreetComplete/issues/2206) ? Or, if something else, can you make a screenshot depicting problem?
